### PR TITLE
[RM Ch09] Add content 9.5.2.2 Software CI/CD Reqts

### PR DESCRIPTION
**Pull request by pgoyal01**
Added "9.5.2.2 Software CI/CD Requirements" from [RM Automation Content](https://wiki.lfnetworking.org/display/LN/RM+Automation+Content)

### CODE REVIEW

**Review by CsatariGergely — CHANGES_REQUESTED** [review]
Can you please also add the source of the figures?

**Review by walterkozlowski — APPROVED** [review]

**Review by karinesevilla — COMMENTED** [review]
@pgoyal01 The storage of images and manifests in repositories does not appear enough in the requirements. Can we add it to auto.cicd.008?
"The CI pipeline must validate images and manifests, and store them in a repository"

File: doc/ref_model/chapters/chapter09.md [hunk 1]
```diff
@@ -192,7 +192,24 @@ As such the requirements for Capacity Management on the infrastructure are cover
...
+
+Ref # | Description | Comments/Notes
+---|---|---
+auto.cicd.001 | The CI/CD pipeline must support deployment on any cloud and cloud infrastructures including different hardware accelerators. | CI/CD pipelines automate CI/CD best practices into repeatable workflows for integrating code and configurations into builds, testing builds including validation against design and operator specific criteria, and delivery of the product onto a runtime environment.<br>Example of an open-source cloud native CI/CD framework is the Tekton project (https://tekton.dev/)
+auto.cicd.002 | The CI/CD pipelines must use event-driven task automation | 
+auto.cicd.003 | The CI/CD pipelines should avoid scheduling tasks | 
+auto.cicd.004 | The CI/CD pipeline is triggered by a new or updated software release being loaded into a repository | The software release cane be source code files, configuration files, images, manifests.<br>Operators may support a single or multiple repositories and may, thus, specify which repository is to be used for these release.<br>An example, of an open source repository is the CNCF Harbor (https://goharbor.io/)
```
Comment (pgoyal01) [inline doc/ref_model/chapters/chapter09.md:202]: @karinesevilla suggested making clear that images and manifests are also stored in the repository. 
```suggestion
auto.cicd.004 | The CI/CD pipeline is triggered by a new or updated software release being loaded into a repository | The software release can be source code files, configuration files, images, manifests, and these are all stored in a repository.<br>Operators may support a single or multiple repositories and may, thus, specify which repository is to be used for the release.<br>An example, of an open source repository is the CNCF Harbor (https://goharbor.io/)
```

Comment (karinesevilla) [reply doc/ref_model/chapters/chapter09.md:202]: @pgoyal01 Good,I agree

**Review by karinesevilla — APPROVED** [review]

**Review by sukhdevkapur — APPROVED** [review]

**Pull request merged**